packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -934,6 +934,15 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     public async stop(): Promise<IRuntimeState> {
         this.verifyNotClosed();
 
+        // It is not very clear how reload works with local changes.
+        // First, summarizing code likely does not work (i.e. read - produced unknown result)
```

Comment:
hot-reload doesn't work right now, so should never be done. It will be removed as part of #4685, so i wouldn't worry about it too much

Reply:
That's fine, but I'd prefer to have a logging error statement here for future. Changing slight comment